geopandas/tests/test_plotting.py

```diff
@@ -469,10 +471,10 @@ def test_linestrings(self):
         # pass through of kwargs
         coll = plot_linestring_collection(ax, self.lines, linestyle='--',
                                           linewidth=1)
-        exp_ls = _style_to_linestring_onoffseq('dashed')
+        exp_ls = _style_to_linestring_onoffseq('--', 1)
```

Comment:
Why this change? It seems to be causing failure on all matplotlib versions prior to 2.0.0.

Reply:
Ah, I changed it to be consistent with the actual plotting on the previous line, but if it doesn't work everywhere I can revert it.